addons/l10n_pt/models/account_tax.py

```diff
@@ -0,0 +1,61 @@
+from odoo import models, fields
+
+l10n_pt_TAX_REGIONS_SELECTION = [
+    ('PT-ALL', 'All Regions'),
+    ('PT', 'Mainland Portugal'),
+    ('PT-MA', 'Madeira'),
+    ('PT-AC', 'Azores'),
+]
+
+l10n_pt_TAX_CATEGORIES_SELECTION = [
+    ('N', 'Normal'),
+    ('I', 'Intermediate'),
+    ('R', 'Reduced'),
+    ('E', 'Exempt'),
+]
+
+# https://info.portaldasfinancas.gov.pt/pt/apoio_contribuinte/Faturacao/Fatcorews/Documents/Tabela_Codigos_Motivo_Isencao.pdf
+L10N_PT_TAX_EXEMPTION_REASONS_SELECTION = [
```

Comment:
In my opinion this exemption list should be configurable data stored in a model and not hardcoded.
That will make it easier to make updates when changes happen.